handsontable/src/renderers/baseRenderer/baseRenderer.js

```diff
@@ -44,10 +68,14 @@ export function baseRenderer(instance, TD, row, col, prop, value, cellProperties
   removeClass(TD, classesToRemove);
   addClass(TD, classesToAdd);
 
-  TD.ariaColIndex = col;
-  TD.setAttribute('tabindex', '-1');
-  TD.setAttribute('role', 'gridcell');
+  // Remove all accessibility-related attributes for the cell to start fresh.
+  removeAttribute(TD, [
+    new RegExp('aria-(.*)'),
+    new RegExp('role')
+  ]);
```

Comment:
I think that this line should be mandatory. Right now, the user can create a renderer that does not extend the functionality of the `baseRenderer`. That would cause a mess with ARIA tags. 

The best place to reset the tags is [here](https://github.com/handsontable/handsontable/blob/feature/dev-issue-539/handsontable/src/3rdparty/walkontable/src/renderer/cells.js#L100).